dbt/redshift/dbt_project.yml

```diff
@@ -0,0 +1,41 @@
+"analysis-paths":
+- "analysis"
+"clean-targets":
```

Comment:
took me a minute to figure this out! i believe dbt put those these to preserve the file structure - the folders are empty but may be used in the future. since we are not using them for now, i'll go ahead and remove them!